### PR TITLE
Registration_page

### PR DESCRIPTION
**Pull request by olugbengs12**
- I created a signup page and styled with CSS.

- I added a CSS file and an image folder.

- I added an Html file.

### CODE REVIEW

**Comment by JohnItoo** [comment]
Hello @olugbengs12 . Nice job so far.
 Please change the arrangement of your  directories to reflect the task your Registration Page submissions belongs to. i.e either your `uxdesign`, `HTML` or `css` task. Thank you.

File: submissions/OwoeyeGbenga/Html_Task_v1.0/style.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+body {
+    background-image: url(image/pexels.jpeg);
+}
+
+.w-full{
+    width: 100%;
+}
+
```
Comment (ToluAkin) [inline submissions/OwoeyeGbenga/Html_Task_v1.0/style.css:18]: Please remove empty spaces here.

File: submissions/OwoeyeGbenga/Html_Task_v1.0/style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    margin: 5px 0 22px 0;
+    display: inline-block;
+    border: none;
+    background: #f1f1f1;
+    border-radius: 3px;
+  }
+ 
+  hr {
+      color: rgb(27, 41, 230);
+  }
```
Comment (ToluAkin) [inline submissions/OwoeyeGbenga/Html_Task_v1.0/style.css:47]: Add a space here

File: submissions/OwoeyeGbenga/Html_Task_v1.0/style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  .loginBtn {
+    width: 15%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    display: inline-block;
+    border: none;
+    background: white;
+    border-radius: 3px;
+    font-size: 16px;
+  }
```
Comment (ToluAkin) [inline submissions/OwoeyeGbenga/Html_Task_v1.0/style.css:73]: Add space after the curly bracket here too.

File: submissions/OwoeyeGbenga/Html_Task_v1.0/style.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  .registerBtn {
+    width: 80%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    display: inline-block;
+    border: none;
+    background:white;
+    border-radius: 3px;
+    font-size: 16px;
+  }
```
Comment (ToluAkin) [inline submissions/OwoeyeGbenga/Html_Task_v1.0/style.css:58]: Same here...

**Comment by ToluAkin** [comment]
Well done @olugbengs12

**Review by ToluAkin — APPROVED** [review]

**Comment by tegaphilip** [comment]
@olugbengs12 good job.

**Review by tegaphilip — APPROVED** [review]

**Pull request merged**